Tzkt.Sync/Protocols/TezosProtocols.cs

```diff
@@ -49,6 +49,7 @@ public static ProtocolHandler GetProtocolHandler(this IServiceProvider services,
                     "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY" => services.GetRequiredService<Proto13Handler>(),
                     "PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg" => services.GetRequiredService<Proto14Handler>(),
                     "PtLimaPtLMwfNinJi9rCfDPWea8dFgTZ1MeJ9f1m2SRic6ayiwW" => services.GetRequiredService<Proto15Handler>(),
+                    "ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK" => services.GetRequiredService<Proto15Handler>(),
```

Comment:
Actually, you are not the first one who wants to run tzkt on a custom/unsupported protocol, so, since it's not only about alpha, but about potentially any protocol, I'd recommend a more generic solution: ability to specify a fallback protocol in the configuration, so people can run tzkt on any protocol they want and specify any protocol handler for it.

So,
1. 
Let's add `"Fallback": null` to the `"Protocols"` section in the `appsettings.json`.
2.
Let's change `TezosProtocols.GetProtocolHandler` signature to 
```cs
public static ProtocolHandler GetProtocolHandler(this IServiceProvider services, int level, string protocol, string fallback = null)
```
and instead of `"ProtoALpha..." => ...,` use this
```cs
_ when fallback is not null => GetProtocolHandler(services, level, fallback),
```
3.
In the `Observer.cs` add
```cs
private string FallbackProtocol;
```
which can be initialized, for example, in the `Observer.ResetState`. Then, pass it to the `scope.ServiceProvider.GetProtocolHandler` in the `Observer.RebaseLocalBranchAsync` and `Observer.ApplyUpdatesAsync`.

Reply:
Thanks @Groxan. Nice way to address many use-cases in one change. I'll do this and update the PR.

Reply:
@Groxan I made the change you suggested, but there were multiple more places I needed to read or pass the fallback protocol. So I just read it in the `GetProtocolHandler` function, if that's fine.

Reply:
@AlirezaHaghshenas you are right, there are some more places, where the `GetProtocolHandler` is called. Actually, I thought it was ok to left `fallback = null` there, due to some technical reasons, but fine, let's do that in your way. However, sorry for the excessive pickiness, it's better to read `Fallback` from the configuration just once, instead of doing that every time before applying/reverting a block, because it's just excessive CPU work :) I'd like to avoid that, especially on mainnet.

Try this:
```cs
static string Fallback = null;

static string GetFallback(IServiceProvider services)
{
    return Fallback ??= services.GetRequiredService<IConfiguration>().GetValue<string>("Protocols:Fallback");
}

public static ProtocolHandler GetProtocolHandler(..., bool fallback = true)
{
    ...
    _ when fallback => GetProtocolHandler(services, level, GetFallback(services), false),
    ...
}
```

Reply:
Wow. You're right. I'll update the PR. Thanks